merlin/dataloader/loader_base.py

```diff
@@ -626,9 +626,14 @@ def _handle_tensors(self, tensors, tensor_names):
         # TODO: use dict for labels as well?
```

Comment:
I think this comment can be updated along with this this change. We can probably remove the TODO and replace with a description of what is going on here.